DataFormats/Common/interface/Handle.h

```diff
@@ -40,17 +40,16 @@ namespace edm {
     Handle(T const* prod, Provenance const* prov);
 
     Handle(std::shared_ptr<HandleExceptionFactory const>&&);
+
+    Handle(Handle&&) = default;
     Handle(Handle const&) = default;
     Handle& operator=(Handle&&) = default;
     Handle& operator=(Handle const&) = default;
-
-    ~Handle();
+    ~Handle() = default;
```

Comment:
Alternatively I we could just remove all five special methods ?

Reply:
I'd remove them.

Reply:
Done.